PurchasesTests/SubscriberAttributes/BackendSubscriberAttributesTests.swift

```diff
@@ -11,7 +11,7 @@ import XCTest
 class BackendSubscriberAttributesTests: XCTestCase {
 
     let appUserID = "abc123"
-    let now = Date()
+    let now = Date(timeIntervalSinceReferenceDate: 1005000)
```

Comment:
this isn't really `now` anymore

Reply:
I think we should rename this one and also add a comment specifying the exact date the number refers to

Reply:
Done.